app/services/foreman_openscap/client_config/ansible.rb

```diff
@@ -16,7 +16,7 @@ def type
       end
 
       def available?
-        defined?(ForemanAnsible)
+        defined?(ForemanAnsible::VERSION)
```

Comment:
Shouldn't this use `Foreman::Plugin.installed?("foreman_ansible")` instead? I mean, we have a Plugin Registry for a reason? ;)

Reply:
I had the impression the plugin may be registered later, when it's engine is processed. And I am also under the impression migrations don't have the whole environment loaded at this point. But let's try and see if that works in migrations.

Reply:
I'm very happy we enden up using our own framework for checking wether our plugin is installed ^_^